### PR TITLE
Systemd

### PR DESCRIPTION
**Pull request by KitaitiMakoto**
Hi,

I fixed systemd-related sentences in document.

Additionally, I added provisioning setting to remove preinstalled Chef and related packages in Vagrantfile at commit https://github.com/droonga/droonga.org/commit/ed44771622489992ae0f4298ae1cf7167989e66a

Could you consider?
Thanks.

### CODE REVIEW

File: _po/ja/tutorial/1.1.2/benchmark/index.po [hunk 1]
```diff
@@ -1224,8 +1224,8 @@ msgid ""
 "(on node0, node1)\n"
 "% sudo droonga-engine-catalog-generate \\\n"
 "    --hosts=node0,node1\n"
-"% sudo service droonga-engine restart\n"
-"% sudo service droonga-http-server restart\n"
+"% sudo systemctl restart droonga-engine\n"
+"% sudo systemctl droonga-http-server\n"
```
Comment (kou) [inline _po/ja/tutorial/1.1.2/benchmark/index.po:1228]: "restart" is missing.

File: _po/ja/reference/1.1.2/command-line-tools/droonga-engine-configure/index.po [hunk 2]
```diff
@@ -198,35 +198,35 @@ msgid ""
...
-" platform specific location by the command line `service droonga-engine start`"
-", even if this option is given."
+" platform specific location by the command line `systemctl start droonga-engin"
+"e`, even if this option is given."
 msgstr ""
 "`--pid-file=PATH`\n"
 ": デーモンとして実行されたプロセスのプロセスIDの出力先ファイルのパス。\n"
-"  ただし、`service droonga-engine start`コマンドで開始されるサービスについては、プロセスIDは常にプラットフォームごとの適切"
-"な位置に出力されます。"
+"  ただし、`systemcctl start droonga-engine`コマンドで開始されるサービスについては、プロセスIDは常にプラットフォームごと"
```
Comment (kou) [inline _po/ja/reference/1.1.2/command-line-tools/droonga-engine-configure/index.po:216]: There is a too much "c".

**Comment by kou** [comment]
Thanks for your work.
I can fix typos after I merge this pull request. Do you fix them or I fix them?

**Comment by KitaitiMakoto** [comment]
Could you?
I've left my laptop.

**Comment by KitaitiMakoto** [comment]
@kou
sorry i missed mention

**Comment by kou** [comment]
I've fixed them.
Thanks.

**Comment by KitaitiMakoto** [comment]
Thank you!

**Pull request merged**